src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2240,9 +2240,9 @@ namespace RuntimeLanguageProcessor {
 				//                        PsyTsatFnPb(Operand(1)%Number, & ! pressure (Pa)
 				//                                    'EMS Built-In Function') )
 			} else if ( SELECT_CASE_var == FuncCpCW ) {
-				ReturnValue = SetErlValueNumber( CPCW( Operand( 1 ).Number ) ); // result => specific heat of water (J/kg-K) = 4180.d0 | temperature (C) unused
+				ReturnValue = SetErlValueNumber( CPCW() ); // result => specific heat of water (J/kg-K) = 4180.d0 | temperature (C) unused
 			} else if ( SELECT_CASE_var == FuncCpHW ) {
```

Comment:
@mjwitte OK, that made it serious.  Sort-of.  Right now, that argument is doing _nothing_.  Removing this argument as @mbadams5 did cleans up the warning, and does not change functionality _at all_.  But...we can leave it as a comment just after the call, so that should it ever get improved, it will be clear what to pass in.  We don't have to transition Erl, which does sound like a huge mess, we just have to leave it exactly as it is.  Users will have to pass in a dummy value, which they were doing anyway.